playwright-e2e/functional/contested/FR_giveAllocationDirections/give_allocation_direction_spec.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test } from '../../../fixtures/fixtures';
+import config from '../../../config/config';
+import { updateCaseInCcd } from '../../../../test/helpers/utils';
+import { contestedEvents } from '../../../config/case_events';
+import { updateCaseWorkerSteps } from '../../helpers/PayloadHelper';
+import { YesNoRadioEnum } from '../../../pages/helpers/enums/RadioEnums';
+import { ExpressPilotHelper } from '../../helpers/ExpressPilotHelper';
+import { giveAllocationGateKeepingTabData } from '../../../data/tab_content/contested/gatekeeping_and_allocation/gatekeeping_and_allocation_tab';
+
+
+async function createAndProcessFormACase(isExpressPilot: boolean): Promise<string> {
+  const caseId = await  ExpressPilotHelper.createCaseWithExpressPilot(
+    config.applicant_solicitor.email,
+    config.applicant_solicitor.password,
+    './playwright-e2e/data/payload/contested/forma/ccd-contested-base.json',
+    'FinancialRemedyContested',
+    'FR_solicitorCreate',
+    isExpressPilot
+  );
```

Comment:
same as https://github.com/hmcts/finrem-ccd-definitions/pull/2470/files#r2024567731 so a similar method can be reused.